projects/extension/src/background/SmoldotMediator.ts

```diff
@@ -32,7 +32,7 @@ export class SmoldotMediator {
     const nextID = ++this.#id;
     /* eslint-disable @typescript-eslint/no-unsafe-member-access */
     message.id = nextID;
-    this.#smoldotClient.send_json_rpc(JSON.stringify(message));
+    this.smoldotClient.send_json_rpc(JSON.stringify(message));
```

Comment:
if smoldotClient is not a private var then there is no reason of existance of  creating and calling sendRpcMessage(message: JsonRpcRequest): number. It can be called straight

Reply:
No because it depends on the internal id counter. I made it public so I could call terminate but in hindsight I think what would have been better would be to expose a wrapping method that calls terminate on the private client. Users of this class shouldn't be touching the inner client